R/ChromBackend.R

```diff
@@ -62,20 +75,37 @@
 #' - `productMzMax`: for SRM data, optional `numeric` with the upper m/z of
 #'   the product's isolation window.
 #'
-#' @param chromVariables For `selectChromVariables()`: `character` with the
-#'     names of the chromatogram variables to which the backend should be
-#'     subsetted.
+#' @section Core Peaks variables:
+#'
+#' Similar to the *core* chromatogram variables, *core* peaks variables
+#' represent  metadata that should be provided by a backend. Each of these
+#' variables should return a value, and if undefined, a missing value (with the
+#' appropriate data type) is returned. The number of values for a peaks
+#' variable in a single chromatogram can vary, from none to multiple, and may
+#' differ between chromatograms.
+#'
+#' The names of peaks variables in the current chromatogram object can be
+#' obtained with the `peaksVariables()` function.
+#'
+#' Each core peaks variable has a dedicated accessor method.
+
```

Comment:
maybe better remove the empty line here and below - not sure if roxygen would properly render/process such documentation (separated by an empty new line instead of a line starting with `#'`